cellpack/bin/clean.py

```diff
@@ -1,16 +1,17 @@
 # cleans the local cache directory
-import shutil
-from cellpack.autopack import CACHE_DIR
 import fire
 import os
+import shutil
+from cellpack.autopack import CACHE_DIR
 
 
-def clean():
+def clean(cache_dir=CACHE_DIR):
```

Comment:
Adding a param makes it much more flexible! We can integrate it to our workflows later if we find it useful